back/dora/core/models.py

```diff
@@ -97,3 +99,63 @@ class Meta:
 
     def __str__(self):
         return f"{self.import_type} - {self.filename} ({self.status})"
+
+
+class ConsentRecord(models.Model):
+    """
+    Enregistrements de consentement immuables pour la conformité RGPD.
+    Chaque action de consentement crée un nouvel enregistrement (jamais de mise à jour).
+    """
+
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+
+    user = models.ForeignKey(
+        User,
+        on_delete=models.SET_NULL,
```

Comment:
Si l'utilisateur est supprimé je ne vois pas de raison de garder ses consentements :
```suggestion
        on_delete=models.CASCADE,
```

Reply:
changé ici https://github.com/gip-inclusion/dora/pull/604/commits/9a8263b5d727ea0f25cc41e3feef5dd1692757d5

c'est possible qu'on doit garder les records au cas où l'utilisateur est supprimé mais `SET_NULL` n'est pas la bonne façon. si besoin, je ferai quelque chose plus malin dans un autre PR